### PR TITLE
add auto generate dubbo admin swagger

### PR DESCRIPTION
**Pull request by 2456868764**
## What is the purpose of the change

add auto generate dubbo admin swagger  which refer to issue Makefile tasks #1107

## Brief changelog
1. make help 
  dubbo-admin-swagger  Generate dubbo-admin swagger docs.
  swagger-install  Download swagger locally if necessary.

2. add swagger tags in mock_rule model and handler.

3. three suggestions
- Use struct  instead of map in json output
- Rule DTO & VO name for consistent usage
- Directly output business data instead of wrapping ,demo code as follow:

```
	c.JSON(http.StatusOK, gin.H{
		"code": 1,
		"data": []model.TagRouteDto{tagRoute},
	})

```

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [ ] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist.
* [ ] Run `mvn clean compile --batch-mode -DskipTests=false -Dcheckstyle.skip=false -Drat.skip=false -Dmaven.javadoc.skip=true to make sure basic checks pass.

### CODE REVIEW

File: cmd/admin/main.go [hunk 1]
```diff
@@ -23,6 +23,13 @@ import (
 	"github.com/apache/dubbo-admin/pkg/admin/services"
 )
 
+// @title           Dubbo-Admin API
+// @version         1.0
+// @description     This is a dubbo-admin swagger ui server.
+// @license.name  Apache 2.0
+// @license.url   http://www.apache.org/licenses/LICENSE-2.0.html
+// @host      127.0.0.1:38080
```
Comment (DMwangnima) [inline cmd/admin/main.go:31]: Port should be set as 38081?

File: hack/swagger/main.go [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+	swaggerFiles "github.com/swaggo/files"
+	ginSwagger "github.com/swaggo/gin-swagger"
+	"swagger/docs"
+)
+
+func main() {
+	docs.SwaggerInfo.Title = "Dubbo-Admin API"
+	docs.SwaggerInfo.Description = "This is a dubbo-admin swagger ui server."
+	docs.SwaggerInfo.Version = "1.0"
+	docs.SwaggerInfo.Host = "127.0.0.1:38080"
```
Comment (DMwangnima) [inline hack/swagger/main.go]: Same as previous section

Comment (2456868764) [reply hack/swagger/main.go]: OK，i will delete it

File: cmd/admin/main.go (hunk 1 above)
Comment (2456868764) [reply cmd/admin/main.go:31]: i just pick 38081 to avoid port conflict,   any suggestion?

**Comment by 2456868764** [comment]
i have updated , pls review it again

File: cmd/admin/main.go (hunk 1 above)
Comment (DMwangnima) [reply cmd/admin/main.go:31]: It's okay. If there is a conflict in the future, let's take it into account.

**Comment by DMwangnima** [comment]
LGTM

**Comment by 2456868764** [comment]
i'v updated swagger struct according to what discussed in weekly meeting last night, pls review it again . If it is ok , pls  merge into main branch.

**Review by chickenlj — APPROVED** [review]
LGTM.

**Pull request merged**